sdk/storage/AzureStorageBlob/Source/TransferManager.swift

```diff
@@ -59,10 +58,24 @@ internal protocol TransferManager {
 
 /// A delegate to receive notifications about state changes for all transfers managed by a `StorageBlobClient`.
 public protocol TransferDelegate: AnyObject {
```

Comment:
We can make this internal now, since we don't expose it in the public API for StorageBlobClient.

Reply:
Done